tests/test_black.py

```diff
@@ -1578,7 +1578,33 @@ def test_reformat_one_with_stdin_filename(self) -> None:
                 mode=DEFAULT_MODE,
                 report=report,
             )
-            fsts.assert_called_once()
+            fsts.assert_called_once_with(
+                fast=True, write_back=black.WriteBack.YES, mode=DEFAULT_MODE
+            )
+            # __BLACK_STDIN_FILENAME__ should have been striped
```

Comment:
```suggestion
            # __BLACK_STDIN_FILENAME__ should have been stripped
```

Reply:
I fixed the two other instances in this file as well